packages/web-api/src/types/request/slacklists.ts

```diff
@@ -0,0 +1,12 @@
+import type { Block, KnownBlock } from '@slack/types';
+
+import type { TokenOverridable } from './common';
+
+export interface SlackListsCreateArguments extends TokenOverridable {
```

Comment:
```suggestion
// https://docs.slack.dev/reference/methods/slackLists.create
export interface SlackListsCreateArguments extends TokenOverridable {
```

✏️ suggestion: For future maintainers, these can be kind to keep!